lang/python/pypi.mk

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2019 Jeffery To
```

Comment:
should there be an email here?
just asking; no preference;

Reply:
I've seen copyright notices with and without an email... Probably better to include it I guess